libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteObserver.kt

```diff
@@ -1,16 +1,30 @@
 package com.mapbox.navigation.core.fasterroute
 
 import com.mapbox.api.directions.v5.models.DirectionsRoute
+import java.util.concurrent.TimeUnit
 
 /**
  * Interface definition for an observer that get's notified whenever the SDK finds a faster route to the destination.
  */
 interface FasterRouteObserver {
 
+    /**
+     * Overridable constant to change the frequency of the fastest route update.
+     */
+    fun restartAfterMillis(): Long = TimeUnit.MINUTES.toMillis(2)
```

Comment:
Is this a `fun`ction intentionally? Instead of a `var` 🤔

Reply:
![Screen Shot 2020-04-16 at 10 36 17 AM](https://user-images.githubusercontent.com/3021882/79488123-1b865380-7fce-11ea-8d86-2dccfbe13a0c.png)

Yeah `var` doesn't give a default in interfaces

Reply:
Yeah was wondering if we could handle the default value in the call-site or even using a custom getter 🤔 

In any case, it seems we have better reasons to use a method so 👍 Could you drop a quick comment clarifying why is preferable to use a method in this case?

Reply:
Talked offline and updated the comment. This is also a designed so the sdk can "pull" new restart intervals. A `var` or a `fun` with a variable would suggest that the rate can be "pushed". 

Canceling a request can be achieved with MapboxNavigation.detachFasterRouteObserver